src/helpers/configHelper.js

```diff
@@ -0,0 +1,33 @@
+const defaults = {
+    DARK_MODE: 'auto',          // String (enabled, disabled, auto)
+    SWITCH_COLORS: false,       // Boolean
+    NOTATION_24H: true,         // Boolean
+    WRAP_HEADER: false,         // Boolean
+    COMMON_PATH: [],            // Array of Strings
+    COLUMNS: [                  // Array of Strings in the order they should appear
+        'Name',
+        'Progress',
+        'ETA',
+        'Download Speed',
+        'Upload Speed',
+        'File Size',
+        'Downloaded',
+        'Uploaded',
+        'Downloading from',
+        'Seeding to'
+    ],
+    SORT_COLUMN: 'Progress',    // String
+    SORT_DIRECTION: 'desc'      // String (asc, desc)
+}
```

Comment:
Perhaps we shouldn't have a default config like this, but rather depend on the original file and force it to always be complete with some validation. That way we don't have to worry about maintaining the same values in multiple places.

Reply:
Or perhaps we should even grab the values from LocalStorage here and merge it correctly so that this becomes the master config object: localstorage, config file, default value.

Reply:
You could certainly add a default config file e.g. config.dev.json and merge that with config.json as long as the output is a complete running config so you don't have to test for undefined.

Reply:
On a second though its probably really better to inline defaults individually since config is an object checking for a non existing property will never throw an unrecoverable exception anyway e.g. `config.WHATEVER || 'the default'`
Let me reverse this...

Reply:
Okay, on a thrid thought i like my version better, i would even go so far to NOT ship a config.json as it would get overwritten on an update, maybe instead a config.json.template (with default values) as a starting point or just point the user to the README 🤔